cmd/kube-scheduler/app/server.go

```diff
@@ -150,6 +150,7 @@ func NewOptions() (*Options, error) {
 		return nil, err
 	}
 
+	o.config.LeaderElection.LeaderElect = true
```

Comment:
seems like defaults should be applied in defaults.go, not here (and options round-tripped through those defaults)

Reply:
i was trying to match what is done in kube-controller ... will fix https://github.com/kubernetes/kubernetes/blob/master/cmd/cloud-controller-manager/app/options/options.go#L52